coding_systems/base/tests/helpers.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from django.test import TestCase
+
+
+class DynamicDatabaseStateNotRecordedError(Exception):
+    pass
+
+
+class DynamicDatabaseTestCase(TestCase):
+    """A special TestCase subclass to facilitate use of
+    a dynamically created database.
+
+    db_alias: name of the database that needs to be accessed, str (required)
+    coding_system: name of the coding system, for generating a temporary
+    database directory, str or None
+    import_data_path: name of the path to import data used, str or None
+    """
+
+    @property
+    def db_alias(self):
+        # The db_alias that will be added temporarily to the DB.
+        raise NotImplementedError(
+            "This test class requires a database alias to be set."
+        )
+
+    coding_system = None
```

Comment:
question: Could this be better named? It isn't necessary, but is used to derive what `expected_db_path` is.

Reply:
I've the tests that use temporary paths into another class, and renamed `coding_system` to be clearer.